samples/algorithms/integer-factorization/Program.cs

```diff
@@ -49,8 +49,11 @@ class VisualizeOptions : CommonOptions
         [Option('g', "generator", Required = true, HelpText = "A coprime to `number` of which the period is estimated")]
         public long Generator { get; set; }
 
-        [Option('r', "resource", Required = false, Default = 0, HelpText = "The resource - CNOT: 0; Measure: 1; QubitClifford: 2; R: 3; T: 4")]
+        [Option('r', "resource", Required = false, Default = 0, HelpText = "The resource - CNOT: 0; QubitClifford: 1; Measure: 2; R: 3; T: 4")]
```

Comment:
Is there any place in the code that needs a matching change?

Reply:
Thanks for flagging this. There is no code affected by this, it was just wrong documentation in the CLI info text.